.github/workflows/provision.yml

```diff
@@ -14,6 +14,13 @@ jobs:
       AzureTenantId: ${{ secrets.AZURETENANTID }}
       AzureSubscriptionId: ${{ secrets.AZURESUBSCRIPTIONID }}
       AzureClientSecret: ${{ secrets.AZURECLIENTSECRET }}
+      AzureAdInstance: ${{ secrets.AZUREADINSTANCE }}

```

Comment:
@hassanhabib - can I update the pipeline secrets accordingly?

Reply:
@AndrewMcClelland certainly. make sure you follow this naming convention.
`DMX-GATEKEEPER-APP-DEV`